Source/WebCore/html/canvas/WebGLRenderingContextBase.cpp

```diff
@@ -5039,7 +5039,7 @@ ExceptionOr<void> WebGLRenderingContextBase::texImageSourceHelper(TexImageFuncti
         if (m_unpackFlipY)
             adjustedSourceImageRect.setY(pixels->height() - adjustedSourceImageRect.maxY());
 
-        auto imageData = std::make_optional<GCGLSpan<const GCGLvoid>>(pixels->data().data(), pixels->data().byteLength());
+        auto imageData = GCGLSpan<const GCGLvoid> { pixels->data() };
```

Comment:
this is more succinctly:
`GCGLSpan<const GCGLvoid> imageData { pixels->data() };`
(by analogue of 
```
Vector<uint8_t> data;
vs
auto data = Vector<uint8_t> { };
```